### ISSUE TITLE
SSL Support for WebFist

### ISSUE THREAD

**Issue by coolaj86** [silverbucket/webfinger.js#13]
If I go to https://webfinger.net/ and search my email it is also broken

https://client.webfinger.net/lookup?resource=coolaj86%40gmail.com

**Comment by silverbucket** [comment]
Yeah, it's because WebFist does not support SSL. Not sure what can be done about this in the client library.

**Comment by silverbucket** [comment]
ping @bradfitz - ever plan to implement SSL support for WebFist?

**Comment by bradfitz** [comment]
Maybe at this point I'll wait for https://letsencrypt.org/ to launch, which is already accepting people into their whitelist. Then I'll do it for sure.

**Comment by silverbucket** [comment]
Ah, great! Yeah I'm waiting patiently for it as well. :)

**Comment by skddc** [comment]
It launched! ;)

**Comment by silverbucket** [comment]
As WebFist seems to have been abandoned, we'll be deprecating it as of v2.8.1
